### ISSUE TITLE
false positive `operator-no-unspaced` 

### ISSUE THREAD

**Issue by evilebottnawi** [kristerkari/stylelint-scss#65]
on this scss:
```scss
a {
  transform: translate(-50%, -$no-ui-slider-height);
}
```
Output:
```shell
44:32  ✖  Expected single space after "-"   scss/operator-no-unspaced
```

**Comment by dryoma** [comment]
Thanks again for reporging, please try 1.3.4.
